### PR TITLE
text_on_image Creator - Web design #136 #636

### PR DESCRIPTION
**Pull request by AnamuddinAhmad**

### CODE REVIEW

File: modules/text_on_image/text_on_image_126_by_anamuddinahmad.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+            <h3>DISCOVER IT</h3>
+            <h3>LATEST POSTS</h3>
+            <h3>ABOUT US</h3>
+        </div>
+
+    </div>
+    
+    <script type="text/javascript" src="vanilla-tilt.js"></script>
+    <script>
+        VanillaTilt.init(document.querySelector("#inner-container-1"), {
```
Comment (NtemKenyor) [inline modules/text_on_image/text_on_image_126_by_anamuddinahmad.html:171]: You can use Pure JavaScript here

File: modules/text_on_image/text_on_image_126_by_anamuddinahmad.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        <div id="inner-container-2">
+            <h3>IT INDUSTRY INNOVATION</h3>
+            <h3>DISCOVER IT</h3>
+            <h3>LATEST POSTS</h3>
+            <h3>ABOUT US</h3>
+        </div>
+
+    </div>
+    
+    <script type="text/javascript" src="vanilla-tilt.js"></script>
```
Comment (NtemKenyor) [inline modules/text_on_image/text_on_image_126_by_anamuddinahmad.html:169]: No need to import the entire library just for the functionality below.

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
This is a very good job but we need you to make some changes:

1) We love and prefer Pure JavaScript above all libraries and frameworks and in most cases we review the code to see if the need for more libraries are needed. We(I) feel you do not need to import an entire library to do that. U can simple use pure JS.

2) Also, if you have to install a library (use a library), it most be placed in the right section/folder. We have a sections for style sheets, scripts and libraries etc

I would advise you do this with pure JavaScript and not a combination of Pure and Vanila.
Thank you for your understanding.

**Review by MaverickDe — APPROVED** [review]
Dear @AnamuddinAhmad ,

I indeed express our sincere gratitude for your recent contribution to our open source project. Your commitment to the project has not only enhanced its functionality but has also demonstrated the spirit of collaboration that makes open source communities thrive.

Congratulations on your exceptional work! Your dedication and expertise have significantly contributed to the success of our project, and we are truly fortunate to have you as part of our community.

I must extend my sincere apologies for the delayed response to your pull request. We deeply appreciate your patience and understanding. As our project continues to grow, we recognize the need to improve the quality and response time for contributions. To achieve this, we have initiated a call for maintainers and reviewers who can help us maintain the high standards of the project and ensure timely responses to pull requests.

Your involvement and insights would be invaluable in this endeavor. If you or anyone you know is interested in taking on a more active role in the project, please don't hesitate to reach out. We believe that with a strong team of maintainers and reviewers, we can enhance both the quality of the project and the speed of our responses.

Additionally, I am pleased to inform you that we have activated the sponsorship button for our project. This is an exciting development that allows supporters like you to contribute to the sustainability and growth of the project. Your continued support, whether through code contributions or sponsorship, is truly appreciated.

Once again, congratulations on your outstanding contribution, and thank you for being an essential part of our open source community. We look forward to your continued collaboration and success together.

Best regards,

**Review by AnamuddinAhmad — COMMENTED** [review]
May i know that what's the reason behind my code why it's not marge?

**Review by AnamuddinAhmad — COMMENTED** [review]
Screenshot 2023-11-16 000235

**Review by MaverickDe — APPROVED** [review]

**Pull request closed without merging**